src/journeys_utils.js

```diff
@@ -894,7 +901,9 @@ journeys_utils.animateBannerExit = function(banner, dismissedJourneyProgrammatic
 		if (!dismissedJourneyProgrammatically) {
 			journeys_utils.branch._publishEvent('branch_internal_event_didCloseJourney', journeys_utils.journeyLinkData);
 		}
+
 		journeys_utils.isJourneyDisplayed = false;
+		setTimeout(function(){ journeys_utils.exitAnimationIsRunning = false; }, journeys_utils.animationSpeed )
```

Comment:
Curious why this uses setTimeout when this is already inside a setTimeout. Seems like you could just assign the variable here.

Reply:
This is the problem itself. I use a timeout because we using and CSS animation to adjust the margins. This animation has to be finished. Unfortunately, this happens after the code block is processed. Therefore I have to use this timeout 👍